src/lib/Form/FormDisplay.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import {ExpensionStep, ExpensionStepper} from '../ExpensionStepper';
+import {Section} from './Section';
+import {Id} from '../types/Id';
+import {IAnswer} from '../types/Answer';
+import {QuestionId, QuestionType} from '../types/Question';
+import {defaultMessages} from '../types/Messages';
+import {useEffect, useState} from "react";
+import formAnswerAction from "./form.action";
+import {FormProps} from "./FormProps";
+import {State} from "./form.reducer";
+import {useFormContext} from "./FormContext";
+
+const FormDisplay = (props: FormProps) =>  {
+
+  const {form, messages = defaultMessages, scrollOffset = 0, dateFormat, lang, maxUploadFileSize, readonly = false,
+    onChange, onUploadFile, onRemoveFile, onSectionEnd, onEnd, firstStepPrevCb } = props
+
+  const [loading, setLoading] = useState(false)
+
+  const {state, dispatch} = useFormContext()
+
+  const {answers}: State = state
+
+  useEffect(() => {
+    initReducerParams();
+    initReducerAnswers();
+  }, [])
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has missing dependencies: 'initReducerAnswers' and 'initReducerParams'. Either include them or remove the dependency array.](https://app.codacy.com/gh/Particeep/form-answer/pullRequest?prid=9527160)
```suggestion
  }, [initReducerAnswers, initReducerParams])
```